include/mgp.hpp

```diff
@@ -650,7 +651,7 @@ class Relationship {
   std::string_view Type() const;
 
   /// @brief Returns an iterable & indexable structure of the relationship’s properties.
-  class Properties Properties() const;
+  mgp::Properties Properties() const;
```

Comment:
From now on, we don't use the Properties class, so merge the master first into this PR.

Reply:
Done